packages/design-system/.lintstagedrc.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
this should be moved to the root directory, git hooks should be repository-level and not package-level

Reply:
moved